### PR TITLE
Add tests and configurations for serializers, models, views, and fixtures

### PR DESCRIPTION
**Pull request by velinamons**
- Minor fix: Make alphabetical imports order and grouping
- Add tests for genre listing, game detail, and toggle favorite functionality using reusable paths dictionary
- Create fixture in conftest.py to generate a JWT access token for a given user
- Add test_game_serializer_output; Add RegisterSerializerTestCase to define a set of tests for different functionalities of the RegisterSerializer
- Add Python package tests into game/catalog; Add test_models.py and enable DB access using pytestmark = pytest.mark.django_db
- Add pytest.ini for configuration; Create conftest.py for reusable fixtures

### CODE REVIEW

File: game_catalog/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    assert str(studio) == "Epic Games"
+
+
+def test_game_str(game):
+    assert str(game) == "Fortnite"
+
+
+def test_user_is_favorite(user, game):
+    user.favorite_games.add(game)
+    assert user.is_favorite(game) is True
```
Comment (NEkropo1) [inline game_catalog/tests/test_models.py]:
```suggestion
    assert user.favorited(game) is True
```
I guess would be sligthly more concise, but its up to you.
Enuse if you change name to make it acroos project.

File: game_catalog/tests/test_views.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    return APIClient()
+
+
+@pytest.fixture
+def paths(game, genre):
+    return {
+        "genre_list": "/api/genres/",
+        "game_detail": f"/api/games/{game.id}/",
+        "toggle_favorite": f"/api/games/{game.id}/toggle_favorite/",
+    }
```
Comment (NEkropo1) [inline game_catalog/tests/test_views.py]: QA should test this way, you should import paths from urls

File: game_catalog/views.py
Comment (NEkropo1) [inline game_catalog/views.py:1]: add endline

File: pytest.ini
Comment (NEkropo1) [inline pytest.ini:1]: add endline

**Review by NEkropo1 — CHANGES_REQUESTED** [review]

**Comment by NEkropo1** [comment]
lifehack:
when you make a pr, first check it yourself with your human eyes before asking someone else to check it. You will fix most or all the issues in code ;)

File: game_catalog/tests/test_views.py (hunk 2 above)
Comment (velinamons) [reply game_catalog/tests/test_views.py]: With routers usage we can only make it this way i think:
@pytest.fixture
def paths(game, genre):
    return {
        "genre_list": reverse("genre-list"),
        "game_detail": reverse("game-detail", kwargs={"pk": game.id}),
        "toggle_favorite": reverse("game-toggle-favorite", kwargs={"pk": game.id}),
    }
  will it be okay?

Comment (velinamons) [reply game_catalog/tests/test_views.py]: Done

2185bc4a67d0084fc65ab40d5f2e194e1213a96c

File: game_catalog/tests/test_models.py (hunk 1 above)
Comment (velinamons) [reply game_catalog/tests/test_models.py]: Done

c828f0ce0f04b621429947330e5da0ce086faf7a

File: game_catalog/views.py
Comment (velinamons) [reply game_catalog/views.py:1]: Done

8e76a004796123ac434f9b71ad50a0b74b799409

File: pytest.ini
Comment (velinamons) [reply pytest.ini:1]: Done

8e76a004796123ac434f9b71ad50a0b74b799409

**Review by NEkropo1 — APPROVED** [review]
GJ!

**Pull request merged**